src/pool/srv_pool.c

```diff
@@ -6859,10 +6859,12 @@ pool_svc_update_map(struct pool_svc *svc, crt_opcode_t opc, bool exclude_rank,
 
 	if (svc->ps_pool->sp_reint_mode == DAOS_REINT_MODE_NO_DATA_SYNC) {
 		D_DEBUG(DB_MD, "self healing is disabled for no_data_sync reintegration mode.\n");
-		if (opc == POOL_EXCLUDE || opc == POOL_DRAIN) {
-			rc = ds_pool_tgt_exclude_out(svc->ps_pool->sp_uuid, &target_list);
+		if (opc == MAP_EXCLUDE || opc == MAP_DRAIN) {
```

Comment:
hmm, why do same for DRAIN? 
user wants to DRAIN it first and then reint it back later?
why cannot be done just by exclude + reint.
I am fine to keep it as it now, seems not really necessary.  Mark it as DOWN for DRAIN also looks not very good.
Or for DRAIN, print an INFO log prompt user that in data sync mode, treat it same as EXCLUDE.

Reply:
Remove.